docker/volumeplugin/plugin.go

```diff
@@ -79,70 +77,96 @@ func (d *RancherStorageDriver) init() error {
 }
 
 func (d *RancherStorageDriver) Create(request volume.Request) volume.Response {
-	logrus.Infof("Docker Create request: %v", request)
+	logRequest("create", &request)
+
+	response := volume.Response{}
+	defer logResponse("create", &response)
 
 	if created, err := d.state.IsCreated(request.Name); err != nil {
-		return volErr(err)
+		response.Err = err.Error()
```

Comment:
The pattern I've followed throughout is to set the error in the response and immediately return. At the top of each top-level function, I added a `defer logResponse(...)` that logs the response - but it always logs at info label. That should be changed to log at error level if an error is present in the response.